tests/data/transformer-module/00-simple-module/authorization-model.json

```diff
@@ -0,0 +1,52 @@
+{
+  "schema_version": "1.2",
+  "conditions": {
+    "a_check": {
+      "expression": "val <= 10",
+      "metadata": {
+        "file": "org.fga",
+        "module": "org"
+      },
+      "name": "a_check",
+      "parameters": {
+        "val": {
+          "type_name": "TYPE_NAME_INT"
+        }
+      }
+    }
+  },
+  "type_definitions": [
+    {
+      "metadata": {
+          "file": "core.fga",
+          "module": "core"
+      },
+      "relations": {},
+      "type": "user"
+    },
+    {
+      "metadata": {
+        "file": "org.fga",
+        "module": "org",
+        "relations": {
+          "member": {
+            "directly_related_user_types": [
+              {
+                "condition": "a_check",
+                "type": "user"
+              }
+            ],
+            "module": "org",
```

Comment:
was thinking we don't include the metadata in relations unless they differ from the type. It would drastically reduce size from some of the bigger models. WDYT?

Reply:
I've no real strong opinion here, I mainly added them to ensure consistency but not having them does have benefits like helping clearly show what is extension and what is not as well as some potential size difference gains, it might even reduce transformation time as we have to loop over all relations to add this specific one.

I'll remove now and if we realise we need them it's easy to add back.

Reply:
Also just as I can't reply to `openfga/api` comment, agreed on removing from `RelationReference` I added there while I was still stumbling around figuring out the right places and just forgot to remove. Have removed from there and updated `js-sdk` accordingly